examples/v1beta1/README.md

```diff
@@ -1,391 +1,141 @@
+[this guide](https://www.kubeflow.org/docs/components/katib/resume-experiment/)
+to know more about it. Check the following examples:
+
+- [Resume From Volume](./resume-experiment/from-volume-resume.yaml)
+
+- [Never Resume](./resume-experiment/never-resume.yaml)
+
+## Metrics Collector
+
+Katib supports the various metrics collectors and metrics strategies.
+Check the [official guide](https://www.kubeflow.org/docs/components/katib/experiment/#configuration-spec)
+to know more about it. In this directory you can find the following examples:
+
+- [File Metrics Collector](./metrics-collector/file-metrics-collector.yaml)
+
+- [Custom Metrics Collector](./metrics-collector/custom-metrics-collector.yaml)
+
+- [Metrics Collection Strategy](./metrics-collector/metrics-collection-strategy.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Settings
+
+You can specify different settings for your Trial template. To know more about it
+follow [this guide](https://www.kubeflow.org/docs/components/katib/trial-template/#use-trial-template-to-submit-experiment).
+Check the following examples:
+
+- [Trial with ConfigMap Source](./trial-settings/trial-configmap-source.yaml)
+
+- [Trial with Metadata Substitution](./trial-settings/trial-metadata-substitution.yaml)
+
+## TODO (andreyvelich) Discuss about this name. Trial Training Containers
+
+Check the following examples for Trial training containers:
+
+- [MXNet MNIST](./trial-training-containers/mxnet-mnist)
+
+- [PyTorch MNIST](./trial-training-containers/pytorch-mnist)
+
+## Katib with Kubeflow Training Operator
+
+Katib has out of the box support for the [Kubeflow Training Operators](https://github.com/kubeflow/tf-operator) to
+perform Trial's [Worker job](https://www.kubeflow.org/docs/components/katib/overview/#trial).
+Check the following examples for the various distributed operators:
+
+- [TFJob MNIST with summaries](./kubeflow-training-operator/tfjob-mnist-with-summaries.yaml)
+
+- [PyTorchJob MNIST](./kubeflow-training-operator/pytorchjob-mnist.yaml)
+
+- [MXJob BytePS](./kubeflow-training-operator/mxjob-byteps.yaml)
+
+- [XGBoostJob LightGBM](./kubeflow-training-operator/xgboostjob-lightgbm.yaml)
+
+- [MPIJob Horovod](./kubeflow-training-operator/mpijob-horovod.yaml)
+
+## Katib with Kubeflow Pipelines
+
+To run Katib with [Kubeflow Pipelines](https://github.com/kubeflow/pipelines) check
+[these examples](./kubeflow-pipelines).
+
+## Katib with Argo Workflows
+
+To know more about using [Argo Workflows](https://github.com/argoproj/argo-workflows)
+in Katib check [this directory](./argo).
+
+## Katib with Tekton Pipelines
+
+To know more about using [Tekton Pipelines](https://github.com/tektoncd/pipeline)
+in Katib check [this directory](./tekton).
+
+## FPGA Support in Katib Experiments
+
+You can run Katib Experiments on [FPGA](https://en.wikipedia.org/wiki/Field-programmable_gate_array)
+based instances. For more information check [these examples](./fpga).
```

Comment:
I think this parts should have one more hierarchy deeper.
Such as `FPGA Support` under `Use cases in Katib Experiments`.
If there came more use cases, like asic placement example, computer vision example, nlp example, ..., current structure leaves all examples in this README.md

Reply:
That's good idea, what do you think @johnugeorge @gaocegege @eliaskoromilas ?

Reply:
Good idea!

Reply:
I think we should decide what is `use-cases`, does Argo and Tekton Experiment also related to the use-cases ?
Maybe we should decide about it in the following issue and keep the structure as it is for now ?

WDYT @johnugeorge @eliaskoromilas @anencore94 ?